Tools/scripts/generate_global_objects.py

```diff
@@ -293,6 +302,16 @@ def generate_runtime_init(identifiers, strings):
                 # since iter_files() ignores .h files.
                 printer.write(f'string = &_Py_ID({i});')
                 printer.write(f'PyUnicode_InternInPlace(&string);')
+        printer.write('')
+        printer.write('#ifdef Py_DEBUG')
+        printer.write("static inline void")
+        with printer.block("_PyStaticObjects_CheckRefcnt(void)"):
+            for i in immortal_objects:
+                with printer.block(f'if (Py_REFCNT({i}) < _PyObject_IMMORTAL_REFCNT)', ';'):
```

Comment:
Why ``<``? Shouldn't the refcount be exactly the same?

Reply:
Refcount can be higher if static types are used as they hold interned strings ex: `__add__`  and are never finalized so we only check for `<`.